libs/vkd3d/swapchain.c

```diff
@@ -389,6 +412,13 @@ static void dxgi_vk_swap_chain_cleanup(struct dxgi_vk_swap_chain *chain)
     for (i = 0; i < ARRAY_SIZE(chain->present.vk_swapchain_fences); i++)
         VK_CALL(vkDestroyFence(chain->queue->device->vk_device, chain->present.vk_swapchain_fences[i], NULL));
 
+    if (chain->queue->device->vk_info.NV_low_latency2)
+    {
+        vkd3d_free(chain->present.low_latency_present_modes);
```

Comment:
Does this really need to be allocated on heap? Could just be a fixed 32-entry array or something.

Reply:
I made it an array of 16 VkPresentModeKHR values for consistency with dxgi_vk_swap_chain_check_present_mode_support.